kogito-timer-quarkus/README.md

```diff
@@ -169,11 +262,22 @@ You need to download the job service and start it locally
 java -Dquarkus.http.port=8085 -jar target/jobs-service-{version}-runner.jar
```

Comment:
Where to download the jobs-service jar ?

https://repo.maven.apache.org/maven2/org/kie/kogito/jobs-service/
and suggest to take the latest version

Reply:
fixed @radtriste